### PR TITLE
Hcpcs refresh 2022

### PR DESCRIPTION
**Pull request by m-khitrun**

### CODE REVIEW

**Review by Alexdavv — APPROVED** [review]
Urgent release for #697
Script to be refactored looking further

**Review by hardhouse — CHANGES_REQUESTED** [review]
leaved for subsequent period

```sql
UPDATE concept_stage cs
SET domain_id = i.domain_id
FROM (
	SELECT DISTINCT ON (crs.concept_code_1) crs.concept_code_1,
		c.domain_id
	FROM concept_relationship_stage crs
	JOIN concept c ON c.concept_code = crs.concept_code_2
		AND c.vocabulary_id = crs.vocabulary_id_2
	WHERE crs.relationship_id = 'Maps to'
		AND crs.invalid_reason IS NULL
		AND crs.vocabulary_id_1 = 'HCPCS'
	ORDER BY crs.concept_code_1,
		CASE c.domain_id
			WHEN 'Drug'
				THEN 1
			WHEN 'Visit'
				THEN 2
			WHEN 'Provider'
				THEN 3
			WHEN 'Device'
				THEN 4
			END
	) i
WHERE cs.concept_code = i.concept_code_1;
```

**Review by TinyRickC137 — APPROVED** [review]

**Review by hardhouse — APPROVED** [review]

File: HCPCS/load_stage.sql [hunk 1]
```diff
@@ -1056,8 +1071,49 @@ FROM t_domains t
...
+	invalid_reason
+	)
+SELECT CASE
+		WHEN c.concept_name LIKE '% (Deprecated)'
+			THEN c.concept_name -- to support subsequent source deprecations
+		WHEN (
+				COALESCE(c.invalid_reason, 'D') = 'D'
+				OR (
+					c.standard_concept = 'S'
+					AND c.valid_end_date < TO_DATE('20991231', 'YYYYMMDD')
```
[the remark this answers is not in the document]
Comment (m-khitrun) [reply HCPCS/load_stage.sql:1092]: Documentation fixed

File: HCPCS/readme.md
[the remark this answers is not in the document]
Comment (m-khitrun) [reply HCPCS/readme.md:1]: Deleted

**Pull request merged**